vaccine_feed_ingest/stages/ingest.py

```diff
@@ -130,12 +135,23 @@ def run_parse(
             check=True,
         )
 
-        if not outputs.data_exists(parse_output_dir):
+        if not outputs.data_exists(
+            parse_output_dir, suffix=STAGE_OUTPUT_SUFFIX[PipelineStage.PARSE]
+        ):
             logger.warning(
                 "%s for %s returned no data files.", parse_path.name, site_dir.name
```

Comment:
Could we add the expected suffix to this error message (to help a developer troubleshoot)?
```suggestion
                "%s for %s returned no data files with expected extension %s.", parse_path.name, site_dir.name, STAGE_OUTPUT_SUFFIX[PipelineStage.PARSE]
```

Reply:
Improved error message here are elsewhere to include extension required